roles/vagrant/tasks/main.yml

```diff
@@ -1,15 +1,23 @@
 ---
-- name: 'install hashicorp repo'
+- name: 'install pvalena/rubygems copr'
   get_url:
-    url: https://rpm.releases.hashicorp.com/RHEL/hashicorp.repo
-    dest: /etc/yum.repos.d/hashicorp.repo
+    url: https://copr.fedorainfracloud.org/coprs/pvalena/rubygems/repo/epel-{{ ansible_distribution_major_version }}/pvalena-rubygems-epel-{{ ansible_distribution_major_version }}.repo
+    dest: /etc/yum.repos.d/pvalena-rubygems.repo
+
+- name: 'install pvalena/vagrant copr'
+  get_url:
+    url: https://copr.fedorainfracloud.org/coprs/pvalena/vagrant/repo/epel-{{ ansible_distribution_major_version }}/pvalena-vagrant-epel-{{ ansible_distribution_major_version }}.repo
+    dest: /etc/yum.repos.d/pvalena-vagrant.repo
```

Comment:
Thoughts about using https://docs.ansible.com/ansible/latest/collections/community/general/copr_module.html?

Reply:
I prefer to stay away from `community.general`

Reply:
Fair enough.